scautable/src/csvIterator.scala

```diff
@@ -30,17 +32,26 @@ import scala.annotation.publicInBinary
   * ```
   * etc
   */
-class CsvIterator[K <: Tuple] @publicInBinary private[scautable] (private val rows: Iterator[String], val headers: Seq[String]) extends Iterator[NamedTuple[K, StringyTuple[K & Tuple]]]:
+class CsvIterator[K <: Tuple, V <: Tuple] @publicInBinary private[scautable] (private val rows: Iterator[String], val headers: Seq[String])(using decoder: RowDecoder[V]) extends Iterator[NamedTuple[K, V]]:
```

Comment:
Does the `decoder: RowDecoder` need to be implicit at this point? It's only use is explicit so I'm wondering if it can't be a normal parameter?